packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -584,3 +583,268 @@ test(`zcf.makeZCFMint - burnLosses - seat exited`, async t => {
     message: `seat has been exited`,
   });
 });
+
+const makeOffer = async (zoe, zcf, proposal, payments) => {
+  let zcfSeat;
+  const getSeat = seat => {
+    zcfSeat = seat;
+  };
+  const invitation = await zcf.makeInvitation(getSeat, 'seat');
+  const userSeat = await E(zoe).offer(invitation, proposal, payments);
+  return { zcfSeat, userSeat };
+};
+
+const sameAsNormalZCFSeat = async (t, emptySeat, normalSeat) => {
```

Comment:
since it's not exhaustive, I'd prefer `likeNormalSeat()` or `similarToNormalSeat()`

Reply:
Done